.github/workflows/get_runner_image.yaml

```diff
@@ -3,22 +3,36 @@ name: Get runner image
 on:
   workflow_call:
     outputs:
+      name:
+        description: "Get first occurrence of build-on base name"
+        value: ${{ jobs.get-runner-image.outputs.name }}
+      channel:
+        description: "Get first occurrence of build-on base channel"
+        value: ${{ jobs.get-runner-image.outputs.channel }}
       runs-on:
-        description: "Get first occurence of build-on base as image runner"
+        description: "Get first occurrence of build-on base as image runner"
         value: ${{ jobs.get-runner-image.outputs.runs-on }}
 
 jobs:
   get-runner-image:
     name: Get runner image
     runs-on: ubuntu-22.04
     outputs:
+      name: ${{ env.BUILD_ON_NAME }}
+      channel: ${{ env.BUILD_ON_CHANNEL }}
       runs-on: ${{ env.BUILD_ON }}
     steps:
       - uses: actions/checkout@v3
       - name: Get build-on value
         run: |
+          name="ubuntu"
+          channel="20.04"
           image="ubuntu-22.04"
           if [ -f charmcraft.yaml ]; then
-            image=$(yq '[.bases.[0].build-on.[0].name, .bases.[0].build-on.[0].channel] | join("-")' charmcraft.yaml)
+            name=$(yq '.bases.[0].build-on.[0].name // "ubuntu"' charmcraft.yaml)
+            channel=$(yq '.bases.[0].build-on.[0].channel // '20.04'' charmcraft.yaml)
+            image="$name-$channel"
```

Comment:
2 questions here:
- If name an channel are "ubuntu" and "20.04" by default, why the default image is "ubuntu-22.04" I was expecting "ubuntu-20.04"
- Depending on the answer to the first question, if image is always name-channel, then I put `image="$name-$channel" outside the `if` statement and would remove the default image value